Dockerfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
```

Comment:
In my opinion it's better to put it in the tests folder because the `hepcrawl_base` image is used for all the tests now, both unit and functional. 
What do you think?